rules/windows/registry_event/sysmon_comhijack_sdclt.yml

```diff
@@ -0,0 +1,25 @@
+title: COM hijack via Sdclt
+id: 07743f65-7ec9-404a-a519-913db7118a8d
+status: experimental
+description: Detects changes to 'HKCU\Software\Classes\Folder\shell\open\command\DelegateExecute'
+references:
+    - http://blog.sevagas.com/?Yet-another-sdclt-UAC-bypass
+    - https://www.exploit-db.com/exploits/47696
+tags:
+    - attack.privilege_escalation
+    - attack.t1546
+    - attack.t1548
+author: Omkar Gudhate
+date: 2020/09/27
+logsource:
+    category: registry_event
+    product: windows
+detection:
+    selection:
+        TargetObject:
```

Comment:
I don't understand the TargetObject selection. 
Isn't a registry access to this key pretty common `HKCU\Software\Classes\Folder\shell\open\command`?

Reply:
Oh, combining folder\shell\command & value as ‘DelegateExecute’ I didnt see much false positives. This needs to be set in order to bypass uac through sdclt execution. Should i add sysmon id 13 in category ? Or add sdclt process execution condition as well?

Reply:
Also i opened up a new duplicate pr for same logic. New to sigma rule creations, might need few more suggestions before getting it right